xtask/codegen/src/language_kind.rs

```diff
@@ -1,4 +1,4 @@
-use crate::css_kinds_src::CSS_KINDS_SRC;
+use crate::{css_kinds_src::CSS_KINDS_SRC, yaml_kinds_src::YAML_KINDS_SRC};
```

Comment:
nit: it seems that we have one line per crate, instead of nested imports.

Reply:
That was the code editor auto importing, fixed.